src/analysis/retail/evoker/augmentation/modules/breahtofeons/BreathOfEonsRotational.tsx

```diff
@@ -227,14 +227,20 @@ class BreathOfEonsRotational extends Analyzer {
      * Due to other Augmentations apparently also wanting to use Breath of Eons alongside you :) */
     const flightEvents = getBreathOfEonsBuffEvents(event);
     const flightData: SpellTracker[] = [];
+    let debuffHits = getBreathOfEonsDebuffApplyEvents(event).length;
+    // if breath was from pre-cast we won't neccesarily have the proper links.
+    // So set this to 1 to make graph look proper
+    if (debuffHits === 0) {
```

Comment:
Will this also be true if you just straight up miss?

Reply:
This is used to decide the size of the area being drawn on the graph for the time spent flying, so doesn't really affect anything besides making sure it actually has a "size" beyond 0.

Reply:
To add to this `temporalWoundsCounter` in `breathPerformance` array is used to determine whether or not a Breath hit.